IFC4x3/Properties/p/PropagationSpeedCoefficient_0AK8i5JmXDxfWTbSTJ3n91/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicates the propagation speed coefficient of feeder.
```

Comment:
This is a bit more specific than the name. Is it okay like this?

Reply:
e8fa2795fee5e68137dc2ddb7aa173ae1ddc12c0

Reply:
Sorry not this commit!